lib/jsdom/living/nodes/CDATASection.idl

```diff
@@ -1,4 +1,3 @@
-[Constructor(optional DOMString data = ""),
```

Comment:
@domenic nice catch! And thanks for the reference.

https://dom.spec.whatwg.org/#interface-cdatasection